obd/OBDCommand.py

```diff
@@ -122,8 +122,22 @@ def __str__(self):
             return "%s: %s" % (self.header + self.command, self.desc)
         return "%s: %s" % (self.command, self.desc)
 
-    def __repr(self):
-        return "OBDCommand(%s, %s)" % (self.name, self.command)
+    def __repr__(self):
+        e = self.ecu
+        if self.ecu == ECU.ALL:
+            e = "ECU.ALL"
+        if self.ecu == ECU.ENGINE:
+            e = "ECU.ENGINE"
+        if self.ecu == ECU.TRANSMISSION:
+            e = "ECU.TRANSMISSION"
+        if self.header == ECU_HEADER.ENGINE:
+            return ("OBDCommand(%s, %s, %s, %s, raw_string, ecu=%s, fast=%s)"
+                    ) % (repr(self.name), repr(self.desc), repr(self.command),
+                 self.bytes, e, self.fast)
+        return (
+            "OBDCommand(%s, %s, %s, %s, raw_string, ecu=%s, fast=%s, header=%s)"
+        ) % (repr(self.name), repr(self.desc), repr(self.command), self.bytes,
```

Comment:
Sorry, I meant to merge this earlier but I just noticed that the indentation doesn't line up. Do you mind fixing it?

Reply:
Done

Reply:
I meant can you line up the brackets, like the previous return statement. Something more like:

```
        return (
            "OBDCommand(%s, %s, %s, %s, raw_string, ecu=%s, fast=%s, header=%s)"
               ) % (repr(self.name), repr(self.desc), repr(self.command),
                    self.bytes, e, self.fast, repr(self.header))
```

Reply:
Indeed I checked my last commit with [yapf](https://github.com/google/yapf) and [autopep8](https://github.com/hhatto/autopep8) and was looking good....